lfx-mentorship/2022/01-Spring/project_ideas.md

```diff
@@ -318,7 +318,7 @@ This internship involves improving the existing Katacoda scenario and adding new
 
 ###### Improvising unit test coverage(CAPV)
 
-- **Description**: Cluster API (CAPI) is a Kubernetes sub-project focused on providing declarative APIs and tooling to simplify lifecycle management of Kubernetes clusters. CAPV is the infrastructure provider that extends Cluster API to manage Kubernetes clusters on vSphere. As a mentee, you will start with learning CAPI/CAPV concepts and then, will work on the main project which is to improve unit test coverage. The ideal percentage is 70%. This project aims to either achieve that or come close to it.
+- **Description**: Cluster API (CAPI) is a Kubernetes sub-project focused on providing declarative APIs and tooling to simplify lifecycle management of Kubernetes clusters. CAPV is the infrastructure provider that extends Cluster API to manage Kubernetes clusters on vSphere. As a mentee, you will start with learning CAPI/CAPV concepts and then, will work on the main project which is to improve unit test coverage. This project aims to either achieve that or come close to it.
```

Comment:
I don't think this sentence makes sense anymore without the prior sentence being removed:

> This project aims to either achieve that or come close to it.